modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -1,26 +1,30 @@
 import { Fun } from '@ephox/katamari';
 
+import { expandRng } from 'tinymce/core/fmt/ExpandRange';
+
 import * as CaretRangeFromPoint from '../../selection/CaretRangeFromPoint';
 import * as NormalizeRange from '../../selection/NormalizeRange';
 import * as RangeCompare from '../../selection/RangeCompare';
 import * as RangeNodes from '../../selection/RangeNodes';
 import { RangeLikeObject } from '../../selection/RangeTypes';
 import * as RangeWalk from '../../selection/RangeWalk';
 import * as SplitRange from '../../selection/SplitRange';
-import DOMUtils from './DOMUtils';
+import Editor from '../Editor';
 
 interface RangeUtils {
   walk: (rng: Range, callback: (nodes: Node[]) => void) => void;
   split: (rng: Range) => RangeLikeObject;
   normalize: (rng: Range) => boolean;
+  expand: (range: Range, options?: { type: 'word' }) => Range;
```

Comment:
The `range` variable name is inconsistent to the rest of this file. We should be trying to keep things consistent, especially in public APIs.

- [ ] Rename `range` to `rng` in the new code for consistency